pkg/adafruitio/adafruitio.go

```diff
@@ -78,6 +78,40 @@ func (c *Client) Record(feed, value string, timestamp time.Time) error {
 	return nil
 }
 
+// GetFeed returns all feeds of the Adafruit user.
```

Comment:
If we _do_ want all feeds (which again, I think we don't), this should be called `Feeds()`.